src/model_context.rs

```diff
@@ -15,40 +15,40 @@ var __initialized = false";
 pub const STATE_METHODS: &'static str = "\
 # Unset a property (as if it was never assigned)
 func unset(property_name) -> void:
-    __assigned_properties.erase(property_name)
+	__assigned_properties.erase(property_name)
```

Comment:
spaces got converted to tabs?

Reply:
It's because we had tabs in the template file, and Godot requires that the whitespace be consistent throughout the file. We could also change it to all spaces?

Reply:
i thought it was already all spaces

Reply:
Seems like godot 4 enforces consistent tabs/spaces throughout the file (it was per-block in godot 3), but it will auto-convert to tabs on save by default. Created an issue to handle indentation automatically, but it's not very critical. We can just make everything tabs for now.

https://github.com/jamie-pate/typescript-to-gdscript/issues/3